Source/WebCore/Modules/encryptedmedia/legacy/LegacyCDMPrivateClearKey.h

```diff
@@ -50,7 +48,15 @@ class LegacyCDMPrivateClearKey : public CDMPrivateInterface {
     bool supportsMIMEType(const String& mimeType) override;
     std::unique_ptr<LegacyCDMSession> createSession(LegacyCDMSessionClient&) override;
 
-protected:
+    void ref() const final { RefCounted::ref(); }
+    void deref() const final { RefCounted::deref(); }
+
+private:
+    explicit LegacyCDMPrivateClearKey(LegacyCDM* cdm)
+        : m_cdm(cdm)
+    {
+    }
+
     LegacyCDM* m_cdm;
```

Comment:
Is this raw pointer safe??

Reply:
It doesn't seem like this pointer is safe given CDMPrivateInterface can now outlive LegacyCDM?

Reply:
I agree. I'll fix that. Thanks for catching!